testing/internal-integ-testing/src/test/groovy/org/gradle/test/fixtures/server/http/BlockingHttpServerTest.groovy

```diff
@@ -1288,20 +1289,29 @@ class BlockingHttpServerTest extends ConcurrentSpec {
         server.start()
 
         when:
-        async {
-            start {
-                succeeds("a")
-            }
-            start {
-                handle.waitForAllPendingCalls()
-                handle.releaseAll()
-                // Should release the request here
+        def thrownException = null
+        try {
+            async {
+                start {
+                    succeeds("a")
```

Comment:
🤔 

This is the place we expect the optional `IOException`, right? Could make the `try` wrap just here.

Alternatively, we could conditionally (if JVM version is >= 24) expect a failure with `fails("a")` instead of `succeeds("a")` and avoid the `try` block altogether.

Reply:
Thanks, I went with the `fails`.